src/components/game/Game.js

```diff
@@ -11,18 +11,25 @@ import ItemTypes from "../../utils/ItemTypes";
 import formatData from "../../utils/formatData";
 import Header from "../header/Header";
 import Main from "../main/Main";
+import {
+    GameFlex,
+    ItemContainer,
+    TopContainer,
+    ScoreText,
+    ExplainText,
+} from "./Game.style";
 
 const Game = ({ data }) => {
     const [score, setScore] = React.useState(0);
-    const [time, setTime] = React.useState(2);
+    const [time, setTime] = React.useState(30);
     const [items, setItems] = React.useState(formatData(data));
     const [showModal, setShowModal] = React.useState(false);
     const [active, setActive] = React.useState(false);
     const [images, setImages] = React.useState([]);
 
     const handleImageLoad = () => {
         setImages(images.concat(true));
-        if (images.length === 9) setActive(true);
+        if (images.length === 5) setActive(true);
```

Comment:
could we set this to say 'items.length -1' so we're not hard-coding this number?